.github/workflows/doc-build.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     - name: build-docs
       run: |
         source zephyr-env.sh
-        make -C doc htmldocs
+        SPHINXOPTS="'-q -j auto'" make -C doc htmldocs
```

Comment:
Do you need all these quotes? Wouldn't `SPHINXOPTS="-q -jauto"` be enough?

Reply:
What about colored output; don't you want to disable it here too?

Reply:
Yes, otherwise you get:
```
cmake -GNinja -B_build -S. -DDOC_TAG=development \
        -DSPHINXOPTS=-q -j auto \
        -DKCONFIG_TURBO_MODE=0
CMake Error: Unknown argument -j
```
instead of
```
cmake -GNinja -B_build -S. -DDOC_TAG=development \
        -DSPHINXOPTS='-q -j auto' \
        -DKCONFIG_TURBO_MODE=0
```
any more elegant suggestion?

Reply:
> What about colored output; don't you want to disable it here too?

Hmm according to help, it auto-detects color availability: `  -N, --no-color    do not emit colored output (default: auto-detect)`, I imagine colored output will not be available on CI.

Reply:
> any more elegant suggestion?

That looks like papering over a bug in the makefile; it should be:

```patch
diff --git a/doc/Makefile b/doc/Makefile
index 12981e465c..08cae1f870 100644
--- a/doc/Makefile
+++ b/doc/Makefile
@@ -23,7 +23,7 @@ htmldocs pdfdocs doxygen: configure
 .PHONY: configure
 configure:
        cmake -GNinja -B${BUILDDIR} -S. -DDOC_TAG=${DOC_TAG} \
-               -DSPHINXOPTS=${SPHINXOPTS} \
+               -DSPHINXOPTS="${SPHINXOPTS}" \
                -DKCONFIG_TURBO_MODE=${KCONFIG_TURBO_MODE}
 
 clean:
```

Reply:
good catch, fixed